python/sglang/srt/managers/scheduler.py

```
@@ -1363,6 +1403,239 @@ def _find_request_by_rid(self, rid: str):
+                new_pool_idx_scalar,
+            )
+        except Exception as e:
+            logger.error(
+                "Failed to inject pending-restore state for rid=%s: %s",
+                rid,
+                e,
+                exc_info=True,
+            )
+            try:
+                mamba_pool.free(new_pool_idx)
+            except Exception as cleanup_error:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)